src/cls/rgw_gc/cls_rgw_gc.cc

```diff
@@ -481,7 +541,7 @@ static int cls_rgw_gc_queue_update_entry(cls_method_context_t hctx, bufferlist *
         } //end - catch
       }
       xattr_urgent_data_map.insert({op.info.tag, op.info.time});
```

Comment:
While we are at it, we can add a check for urgent_data.num_xattr_urgent_entries before getting the xattr urgent data in the update entry call also. let me know if you would want me to add it as a separate commit.